iocore/net/QUICNetVConnection.cc

```diff
@@ -284,8 +284,10 @@ QUICNetVConnection::start()
   this->_frame_dispatcher->add_handler(this->_path_validator);
   this->_frame_dispatcher->add_handler(this->_handshake_handler);
 
+  this->_ack_frame_manager.set_max_ack_delay(params->max_ack_delay_out());
+
   ink_release_assert(this->thread != nullptr);
-  this->_ack_manager_periodic = this->thread->schedule_every(this, params->max_ack_delay_in(), QUIC_EVENT_ACK_PERIODIC);
+  this->_ack_manager_periodic = this->thread->schedule_every(this, params->max_ack_delay_out(), QUIC_EVENT_ACK_PERIODIC);
```

Comment:
However the original code has the same bug, we need to use different settings for incoming connection and outgoing connection.